internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -750,7 +750,16 @@ func convertNetworkConfig(network manifest.NetworkConfig) template.NetworkOpts {
 		AssignPublicIP: template.EnablePublicIP,
 		SubnetsType:    template.PublicSubnetsPlacement,
 	}
-	opts.SecurityGroups = network.VPC.SecurityGroups.GetIDs()
+	mftSecurityGroups := network.VPC.SecurityGroups.GetIDs()
+	var tplSecurityGroups []template.SecurityGroup
+	for _, sg := range mftSecurityGroups {
```

Comment:
nit: these are just recommendations, not blockers!

1. For functions that are of the form `transformXToY` I like naming the variables `in` and `out`, I feel like it makes it pretty clear which one is the transformed version.
2. If the size of the slice is known ahead of time, consider using `make([]T, len(in))` for performance reasons.

```suggestion
	inSGs := network.VPC.SecurityGroups.GetIDs()
	outSGs :=  make([]template.SecurityGroup, len(inSGs))
	for i, sg := range inSGs {
		outSGs[i] = //...
```